languages/python/exercises/concept/dicts/.meta/example.py

```diff
@@ -0,0 +1 @@
+# This file is still empty, I will finish it later. :)
```

Comment:
Add a TO DO as a comment.  That way, the automation will pick it up (_as I understand it_) and add in issues for the missing bits.  😄